packages/x-components/tests/e2e/cucumber/next-queries/next-queries.spec.ts

```diff
@@ -18,14 +20,22 @@ Given(
         }
       }
     };
-    cy.visit('/test/next-queries', {
-      qs: {
-        xModules: JSON.stringify(config)
-      }
-    });
   }
 );
 
+And('next queries response being mock {string}', (mockName: string) => {
+  cy.intercept('https://api.empathy.co/nextQueries', async req => {
+    const module = await import('./stubs/next-queries.stub');
```

Comment:
The dynamic imports idea was made so we could have one stub per file. Some responses like the search one are going to have big stubs, and having multiple in a single file is going to be ugly. But if you are going to do it this way, it only adds unnecessary complexity. So either use dynamic imports with multiple files, or just import the whole module using a standard import.

Reply:
Given that NQs are now static, and how complex it seems to be to allow using dynamic imports inside a Cypress spec file, we can just resolve statically the nqs stub.